src/models/units.ts

```diff
@@ -0,0 +1,52 @@
+import { Unit } from "../authoring";
+
+interface UnitDefinition {
+  title: string;
+  populations: {
+    title: string;
+  };
+  organism: {
+    title: string;
+  };
+  breeding: {
+    title: string;
+  };
+  dna: {
+    title: string;
+  };
+}
+
+type Units = Record<Unit, UnitDefinition>;
+
+export const units: Units = {
+  mouse: {
+    title: "Deer Mice",
+    populations: {
+      title: "Explore: Population",
+    },
+    organism: {
+      title: "Explore: Organism",
+    },
+    breeding: {
+      title: "Explore: Nesting Pairs",
+    },
+    dna: {
+      title: "Explore: DNA",
+    },
+  },
+  pea: {
+    title: "Peas",
+    populations: {
+      title: "Explore: Population",
+    },
+    organism: {
+      title: "Explore: Population",
```

Comment:
I assume this is meant to be a different string, but will probably be changed once the pea work is underway.